clang/test/CodeGenCXX/template-param-objects-nonzero-default-address-space.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang_cc1 -triple spirv64-unknown-unknown -fsycl-is-device -std=c++20 %s -emit-llvm -o - | FileCheck %s
+
+struct S { char buf[32]; };
```

Comment:
Same here, and I can see that the two copies are out-of sync. Would be good to merge the tests.